### ISSUE TITLE
Native module for Flipper seems unavailable

### ISSUE THREAD

**Issue by gabimoncha** [jakobo/expo-community-flipper#10]
Hi, I've just added the plugin to expo config and I get this warning on launch.

```
The native module for Flipper seems unavailable. Please verify that `react-native-flipper` is installed as yarn dependency to your project and, for iOS, that `pod install` is run in the `ios` directory.
at node_modules/react-native-flipper/index.js:158:4 in printNoFlipperWarning
at node_modules/react-native-flipper/index.js:129:25 in addPlugin
at node_modules/@react-navigation/devtools/src/useFlipper.tsx:55:13 in React.useEffect$argument_0
```

**Comment by thecodedrift** [comment]
Thanks for reporting. I think the problem is a missing `post_install` that was added to the podfile. https://fbflipper.com/docs/getting-started/react-native-ios/

To confirm, this is for an iOS build, correct? I'll have a PR out that should address this issue.

**Comment by thecodedrift** [comment]
Take a look at the changes in #11, as I believe that'll address this issue. The expo template seems to have fallen out of sync with the official React Native podfile, as there's now a flipper post install step: https://fbflipper.com/docs/getting-started/react-native-ios/

Based on the RN docs, the additional lines should be what we need.

**Comment by gabimoncha** [comment]
@jakobo will take a look

**Comment by thecodedrift** [comment]
@gabimoncha just released `45.1.0` which should fix this. There was a missing post install step that should have everything working as expected.
